packages/client/src/utils/connection.ts

```diff
@@ -55,6 +55,11 @@ export type SessionConfig = {
   };
   customLlmExtraBody?: any;
   dynamicVariables?: Record<string, string | number | boolean>;
+  connectionDelay?: {
+    other: number;
+    android: number;
+    ios: number;
+  };
```

Comment:
I'd probably expect there to be an option to set all of them at once, but not important and would likely cause rest of the code to be more complicated. Just an idea...

```suggestion
  connectionDelay?: {
    default: number;
    android?: number;
    ios?: number;
  };
```

Reply:
Good point, added